README.md

```diff
@@ -49,10 +49,12 @@
 
```

Comment:
![image](https://github.com/addisalemseifu/library-app-ruby/assets/93039359/b14d28d3-c0b7-479b-b9b6-fda8715e5471)

- [ ] Can you kindly remove the Live Demo section since you don't have any working live link for your app 🤔 Once you have a working live link then you can add this LIve Dem section containing the live link :+1: